### ISSUE TITLE
use in browser

### ISSUE THREAD

**Issue by Level0r0s** [bulentv/js_zklib#59]
Hello! Is it possible to use it directly in the browser?
I need to make the communication using my cell phone and the Browser.
After doing the Record Collection I will send it to a php server using Ajax.

Do you have an example?
Thank you very much

**Comment by ohmmee** [comment]
no, you can't even ping your home network from your browser code.
you'll need processes running in your system that can access file system and do home network communication

**Comment by derit** [comment]
you need a nodejs or alternative use electron
